### PR TITLE
feature/MAMS-30-search-by-folder-status

### PR DESCRIPTION
**Pull request by KatBre**

### CODE REVIEW

File: src/main/java/pl/sdaproject/medicalarchivemanagementsystem/service/LocationService.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import pl.sdaproject.medicalarchivemanagementsystem.model.Location;
 import pl.sdaproject.medicalarchivemanagementsystem.repository.LocationRepository;
 
+import java.util.Collections;
```
Comment (klezner) [inline src/main/java/pl/sdaproject/medicalarchivemanagementsystem/service/LocationService.java]: Delete

File: src/main/java/pl/sdaproject/medicalarchivemanagementsystem/service/LocationService.java [hunk 2]
```diff
@@ -42,7 +43,7 @@ public Location editLocation(Long id, String roomNumber, Integer floor) {
 
         location.setFloor(floor);
         location.setRoomNumber(roomNumber);
-        location.setFolders(folderService.fetchFolder(id));
+        location.setFolders(Collections.singleton(folderService.fetchFolder(id)));
```
Comment (klezner) [inline src/main/java/pl/sdaproject/medicalarchivemanagementsystem/service/LocationService.java]: Delete

Comment (KatBre) [reply src/main/java/pl/sdaproject/medicalarchivemanagementsystem/service/LocationService.java]: done

File: src/main/java/pl/sdaproject/medicalarchivemanagementsystem/service/LocationService.java (hunk 1 above)
Comment (KatBre) [reply src/main/java/pl/sdaproject/medicalarchivemanagementsystem/service/LocationService.java]: done

**Review by klezner — APPROVED** [review]

**Pull request merged**